harvesting-toolkit/README.md

```diff
@@ -3,24 +3,43 @@
 This package includes a general purpose toolkit for archiving open data. We have [quick snippets and code examples](https://github.com/datarefugephilly/workflow/blob/master/harvesting-toolkit/REFERENCE.md) as well as [helpful tools](https://github.com/datarefugephilly/workflow/blob/master/harvesting-toolkit/REFERENCE.md) for quick reference.
 
 
-## Getting set up as a Data Harvester
-- Talk to your DataRescue guide to make sure you can become a Data Harvester.
-- Get set up with the application http://drp-upload.herokuapp.com/
-   - Credentials and other details will be provided by the organizers.
-- Data Harvesters should start by reading this document, which outlines the steps for constructing a proper data archive of the highest possible integrity. The primary focus of this document is on _semi-automated harvesting as part of a team_, and thje workflow described is best-suited for volunteers working to preserve small and medium-sized collections. Where possible, we try to link out to other options appropriate to other circumstances.
-- Contact your DataRescue guide, if you need any assistance.
+## Getting set up as a Harvester
+  - Apply to become a Harvester 
+    - By asking your DataRescue guide or by filling out [this form](XXX) 
+    - Skills recommended: in general, Harvesters need to have some tech skills and a good understanding of harvesting goals.
+    - Note that a Gmail email address is required to apply.
+  - Credentials, slack invite, Uncrawlable Action spreadsheet URL, and other details will be provided once your application is approved.
+  - Test the Uploader application http://drp-upload.herokuapp.com with the credentials provided
```

Comment:
I'm having a hard time understanding the application, could that maybe be explained in the PR comments?